Automattic/vip-go-mu-plugins
# Add/jetpack 10.4

## Description
Added Jetpack 10.4 subtree

## Changelog Description
Jetpack 10.4 has been added to Must-Use plugins. It can be pinned by adding or updating the following line in vip-config.php:
`define( 'VIP_JETPACK_PINNED_VERSION', '10.4');`

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test
1. Check out PR.
1. Add or update pinned version in vip-config.php to be: `define( 'VIP_JETPACK_PINNED_VERSION', '10.4');` 
1. Go to `wp-admin` > `Jetpack`
1. verify the version is 10.4
